adapters/onebot/src/utils.ts

```diff
@@ -252,6 +249,18 @@ export async function adaptSession(bot: BaseBot, data: OneBot.Payload) {
         session.type = 'onebot'
         session.subtype = 'channel-destroyed'
         break
+      case 'offline_file':
+        session.elements = [h('file', data.file)]
```

Comment:
```suggestion
        session.elements = [h.file(data.file)]
```

h 二参数应该是 attr 或者 children，你这个 data.file 是 string 吧？

Reply:
`data.file` is object, [here is it's type](https://github.com/Seidko/satori/blob/17d1e7bad3e3d6f09310efe13139eed746d394c6/adapters/onebot/src/types.ts#L401C1-L408):
```ts
export interface Payload extends Message {
  // ...
  file: File
}

export interface File {
  name: string
  size: number
  url: string
}
```